x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -430,6 +487,36 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
               data-test-subj="mlAnalyticsCreateJobFlyoutDestinationIndexInput"
             />
           </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.excludedFieldsLabel', {
+              defaultMessage: 'Excluded fields',
+            })}
+            helpText={i18n.translate('xpack.ml.dataframe.analytics.create.excludedFieldsHelpText', {
+              defaultMessage:
+                'Optionally select fields to be excluded from analysis. All other supported fields will be included',
+            })}
+          >
+            <EuiComboBox
```

Comment:
Can we clear this if the user changes the source index? That seems to be the behavior with the dependent variable input, which is cleared when the source index changes.

Reply:
Yes, that should be the behavior. Reset when the sourceIndex changes change made here: 24bdeb3e4dbf7b711cb9f257118c356b6baa8c38